docassemble/MLHMotionRegardingChangeOfDomicile/data/questions/review.yml

```diff
@@ -194,20 +177,33 @@ review:
       % endif
 
     css class: bg-secondary-subtle
-    show if: exhibit_attachment.exhibits.has_exhibits
-  # - note: |
-  #     **Uploaded Documents**
+    show if: domestic_violence and exhibit_attachment.exhibits.has_exhibits
+  - Edit:
+      - moving_date
+    button: |
+      **What is the date you want to move?**:
+
+      ${ moving_date }
+  - Edit:
+    - wants_parenting_time_change
+    button: |
+      **Changes to Parenting Time**:
 
-  #     ${ exhibit_attachment.exhibits.add_action() }
+      % if wants_parenting_time_change:
+      You are **asking to change** the parenting time order as follows:
 
-  #     % if exhibit_attachment.exhibits.there_are_any:
-  #     ${ collapse_template(exhibit_attachment.exhibits.in_progress_exhibits) }
-  #     % else:
-  #     You have not uploaded any documents yet.
-  #     % endif
+      ${ requested_order }
 
-  #   css class: bg-secondary-subtle
-  #   show if: exhibit_attachment.exhibits.has_exhibits
+      % if parenting_time_change_agreement == "Yes":
+      **${ other_parties[0] } agrees** with the new parenting time schedule.
+      % elif parenting_time_change_agreement == "No":
+      **${ other_parties[0] } doesn't agree** with the new parenting time schedule.
+      % elif parenting_time_change_agreement == "I don't know":
+      I **don't know if ${ other_parties[0] } agrees** with the new parenting time schedule.
```

Comment:
Suggest changing "I" to "You" to be consistent with other language on the PT review screen.

Reply:
Fixed!